src/renderer/components/app.tsx

```diff
@@ -20,57 +20,57 @@
  */
 import React from "react";
 import { observable, makeObservable } from "mobx";
+import { webFrame } from "electron";
```

Comment:
Did you shuffle order of imports by accident? Quite hard to say what's going on with this file.

Reply:
Yes, will revert.